csrc/runtime/fusion_kernel_runtime.cpp

```diff
@@ -842,6 +842,7 @@ void FusionKernelRuntime::compileKernel(
 
   if (hic != nullptr) {
     auto ke = std::make_unique<KernelExecutor>();
+    ke->setGroupId(group_id);
```

Comment:
so https://github.com/NVIDIA/Fuser/blob/f6792dbc62fd971ac32aff3625285b43e1f81594/csrc/runtime/executor.cpp#L989 picks up the right group ID. Otherwise it defaults to 0, which I think should really be -1.